frontend/src/app/components/ProfileRatingGraph/index.js

```diff
@@ -22,7 +22,7 @@ const SimpleProfileRatingGraph = props => (
       <FlexibleWidthXYPlot
         height={300}
         xType='time'
-        margin={{ left: 45 }}
+        margin={{ left: 45, bottom: 65 }}
```

Comment:
This was done because by the time of former edits, month names weren't truncated and `September` and `November` weren't printed corretly. I removed that [here](https://github.com/salsita/foosball-rating/pull/114/commits/3d398530cdf75f980e6344e885e4f18dd919311a#diff-7f8fb954a84804e36f1c0e7e6293ddf3) as I see month names are truncated now.